exposure-notification/src/main/java/fi/thl/covid19/exposurenotification/configuration/v2/ExposureConfigurationV2.java

```diff
@@ -28,6 +28,12 @@
     public final Map<Integer, String> daysSinceOnsetToInfectiousness;
     public final String infectiousnessWhenDaysSinceOnsetMissing;
 
+    // End of life configurations
+    public final boolean endOfLifeReached;
+    public final Map<Integer, String> endOfLifeStatisticsFI;
+    public final Map<Integer, String> endOfLifeStatisticsSV;
+    public final Map<Integer, String> endOfLifeStatisticsEN;
```

Comment:
Should the key also be a string, just in case we want to display values like "80%"